tests/antares/services/local_services/test_area.py

```diff
@@ -1174,3 +1174,12 @@ def test_translation_txt_is_empty_by_default(self, area_fr, fr_load):
 
         # Then
         assert actual_file_contents == expected_file_contents
+
+
+class TestReadArea:
+    def test_read_areas_local(self, local_read_study):
+        local_area_service = local_read_study.area_service
```

Comment:
You shouldn't access the service directly, you should use the function from the study and the study should call the function depending on which service it has:

```python
local_read_study.read_areas()
```